### ISSUE TITLE
Unknown Error: Undefined index when using command line

### ISSUE THREAD

**Issue by nathan-fiscaletti** [nathan-fiscaletti/synful#195]
Example when running `./synful -migrate`

```
root@vagrant:/var/www/html# ./synful -migrate down
[SYNFUL] [INFO] Migrating CreateApiKeysTable... [SUCCESS]
[SYNFUL] [INFO] Migrating CreateIpFirewallTable... [SUCCESS]
[SYNFUL] [ERRO] Unknown Error: Undefined index: migrate in /var/www/html/vendor/nafisc/parameterparser/src/ParameterParser/ParameterParser.php at line 135
{"error_code":1002,"error":"Either you do not have mod-rewrite enabled or there is no request handler defined in your request."}
```

This has something to do with ParameterParser's `parameter_result_halt()` function and warrants an investigation.

**Comment by nathan-fiscaletti** [comment]
It seems to only happen when you use the alias instead of the name for the command.

**Comment by nathan-fiscaletti** [comment]
This is an issue with the ParameterParser library and will need to me fixed there before we can update here.

**Comment by nathan-fiscaletti** [comment]
Fixed in PR #200
